deploying/deploying.md

```diff
@@ -193,4 +193,4 @@ All done! Small differences in the way you've set up your site may make bits of
 
 - [Tutorial - Heroku Dev Center Deployment](https://devcenter.heroku.com/articles/deploying-nodejs)
 - [Overview of Deployment Options - MDN Express & Node Deployment](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/deployment)
-- [Code School Tutorial(requires sign-in) - Domain Names,Name Servers and Cloud Based Servers](https://www.codeschool.com/beginners-guide-to-web-development/deploying-your-first-website)
+- [Code School Tutorial(requires sign-in) - Domain Names, Name Servers and Cloud Based Servers](https://www.pluralsight.com/paths/html5)
```

Comment:
You have only changed the link but forgot to change the Label.

Reply:
Can you suggest a proper label name for it?

Reply:
`HTML5`. And change the description too.

Reply:
Please only provide free courses.  I think this one requires a membership, plus I don't think HTML5 it is a good replacement for  a tutorial about domain names and servers.

Reply:
Reversed the changes again

Reply:
@alodahl The link was of HTML5 only!

Reply:
Yes, it is wrong and outdated.  Please replace it with a comparable tutorial or delete the whole line.